ci.cue

```diff
@@ -371,13 +371,29 @@ dagger.#Plan & {
 						source: "aiops/"
 						dest: "."
 					},
+					docker.#Run & {
+						command: {
+							name: "pip"
+							args: ["install", "protobuf==3.19.4"]
```

Comment:
shouldn't this be added to requirements.txt?

Reply:
Yes, have added it to the requirements.txt file.

Reply:
This should run `pip install -r requirements.txt`